### ISSUE TITLE
Partials don't support default extension when the view context is a mailer

### ISSUE THREAD

**Issue by cedx** [cedx/yii2-mustache#9]
Using this application configuration :

```php
<?php return [
  'components' => [
    'mailer' => [
      'class' => 'yii\swiftmailer\Mailer',
      'view' => [
        'class' => 'yii\web\View',
        'defaultExtension' => 'mustache',
        'renderers' => ['mustache' => 'yii\mustache\ViewRenderer']
      ]
    ]
  ]
];
```

I expect that I can omit the file extension from partials:

```html
{{> @app/views/header }}
```

But this triggers a `yii\base\InvalidCallException` in `yii\mustache\Loader` class:

```
The view file "[...]/views/header.php" does not exist.
```

(note that the file extension is set to `.php`)

This issue does not appear in other view contexts (e.g. a controller)

**Comment by cedx** [comment]
Yes, the Loader class provides its own implementation of yii\base\View::findViewFile method, and this implementation does not handle the view context (i.e. controller or mailer).

**Comment by cedx** [comment]
Fixed in version 10.0.0 (to be released in the next days).
